### ISSUE TITLE
"Package not found" error for package that exists.

### ISSUE THREAD

**Issue by marjeta42** [meanmail-dev/requirements#82]
**Describe the bug**
Inspection says that package `fastapi` can not be found. 

But running `pip install` from a terminal finds it without a problem. Also right-click + `install all packages` installs it without problem.

All other packages appear fine.

**To Reproduce**
1. Create file `requirements.txt`
2. Add line `fastapi==0.73.0`
3. Observe error

**Expected behavior**
There should be no error for the line `fastapi==0.73.0`.
Also, right-clicking on that line should give the option to install/upgrade/uninstall `fastapi`

**Screenshots**
![image](https://user-images.githubusercontent.com/64432302/165586268-d05481ec-bec4-4d4a-b09b-fd14fc805d4f.png)
![image](https://user-images.githubusercontent.com/64432302/165585855-fb7828a7-92de-4949-9a97-a517a11baeec.png)

**IDE (please complete the following information. See Main Menu -> Help -> About):**
 - OS Mac OS X(x86_64) v12.3.1
 - Product: PyCharm 2022.1 (Professional Edition) Build #PY-221.5080.212, built on April 12, 2022
 - Version 11.0.14.1+1-b2043.25 x86_64
 - JRE Version: VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
 - Project SDK: Python 3.8.12   (also with 3.7 and 3.9)

PY-221.5080.212, JRE 11.0.14.1+1-b2043.25x64 JetBrains s.r.o.

**Plugin (please complete the following information. See Main Menu -> File -> Settings -> Plugins):**
 - Version 2022.3-2022.1

**Comment by josecsotomorales** [comment]
Hey team, also getting this issue, only happens with `fastapi` package:
<img width="382" alt="Screen Shot 2022-04-29 at 5 54 00 PM" src="https://user-images.githubusercontent.com/16229141/166067788-b722c7bf-d98e-43f3-81c5-df377331cfec.png">

**Comment by obrizan** [comment]
For me it happens for several packages, among them: geoalchemy2, transliterate, PyQt5, sphinx, qgis-stubs, types-requests, types-PyYAML. Here is an example:

<img width="330" alt="image" src="https://user-images.githubusercontent.com/929744/166096294-4bbd64a0-bd0c-4e7b-a975-47479365a09b.png">

PyCharm 2022.1 (Professional Edition)
Build #PY-221.5080.212, built on April 12, 2022
Runtime version: 11.0.14.1+1-b2043.25 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 11.6.5

Plugin version:
ru.meanmail.plugin.requirements (2022.3-2022.1)

**Comment by sskorol** [comment]
Same here for `fastapi`. Wondering why no one reported an IDE internal error caused by the `Requirenments` plugin.

```shell
kotlinx.serialization.json.internal.JsonDecodingException: Unexpected JSON token at offset 189434: Expected string literal but 'null' literal was found.
Use 'coerceInputValues = true' in 'Json {}` builder to coerce nulls to default values.
JSON input: .....l}],"0.75.1":[{"comment_text":null,"digests":{"md5":"c61ad39.....
	at kotlinx.serialization.json.internal.JsonExceptionsKt.JsonDecodingException(JsonExceptions.kt:24)
	at kotlinx.serialization.json.internal.JsonExceptionsKt.JsonDecodingException(JsonExceptions.kt:32)
	at kotlinx.serialization.json.internal.AbstractJsonLexer.fail(AbstractJsonLexer.kt:524)
	at kotlinx.serialization.json.internal.AbstractJsonLexer.unexpectedToken(AbstractJsonLexer.kt:202)
	at kotlinx.serialization.json.internal.StringJsonLexer.consumeNextToken(StringJsonLexer.kt:74)
	at kotlinx.serialization.json.internal.StringJsonLexer.consumeKeyString(StringJsonLexer.kt:85)
	at kotlinx.serialization.json.internal.AbstractJsonLexer.consumeString(AbstractJsonLexer.kt:310)
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeString(StreamingJsonDecoder.kt:256)
	at kotlinx.serialization.encoding.AbstractDecoder.decodeStringElement(AbstractDecoder.kt:58)
	at ru.meanmail.pypi.serializers.FileInfo$$serializer.deserialize(FileInfo.kt:5)
	at ru.meanmail.pypi.serializers.FileInfo$$serializer.deserialize(FileInfo.kt:5)
	at kotlinx.serialization.json.internal.PolymorphicKt.decodeSerializableValuePolymorphic(Polymorphic.kt:59)
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:36)
	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableValue(AbstractDecoder.kt:43)
	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableElement(AbstractDecoder.kt:70)
	at kotlinx.serialization.encoding.CompositeDecoder$DefaultImpls.decodeSerializableElement$default(Decoding.kt:535)
	at kotlinx.serialization.internal.ListLikeSerializer.readElement(CollectionSerializers.kt:80)
	at kotlinx.serialization.internal.AbstractCollectionSerializer.readElement$default(CollectionSerializers.kt:51)
	at kotlinx.serialization.internal.AbstractCollectionSerializer.merge(CollectionSerializers.kt:36)
	at kotlinx.serialization.internal.AbstractCollectionSerializer.deserialize(CollectionSerializers.kt:43)
	at kotlinx.serialization.json.internal.PolymorphicKt.decodeSerializableValuePolymorphic(Polymorphic.kt:59)
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:36)
	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableValue(AbstractDecoder.kt:43)
	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableElement(AbstractDecoder.kt:70)
	at kotlinx.serialization.encoding.CompositeDecoder$DefaultImpls.decodeSerializableElement$default(Decoding.kt:535)
	at kotlinx.serialization.internal.MapLikeSerializer.readElement(CollectionSerializers.kt:111)
	at kotlinx.serialization.internal.MapLikeSerializer.readElement(CollectionSerializers.kt:84)
	at kotlinx.serialization.internal.AbstractCollectionSerializer.readElement$default(CollectionSerializers.kt:51)
	at kotlinx.serialization.internal.AbstractCollectionSerializer.merge(CollectionSerializers.kt:36)
	at kotlinx.serialization.internal.AbstractCollectionSerializer.deserialize(CollectionSerializers.kt:43)
	at kotlinx.serialization.json.internal.PolymorphicKt.decodeSerializableValuePolymorphic(Polymorphic.kt:59)
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:36)
	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableValue(AbstractDecoder.kt:43)
	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableElement(AbstractDecoder.kt:70)
	at ru.meanmail.pypi.serializers.PackageInfo$$serializer.deserialize(PackageInfo.kt:5)
	at ru.meanmail.pypi.serializers.PackageInfo$$serializer.deserialize(PackageInfo.kt:5)
	at kotlinx.serialization.json.internal.PolymorphicKt.decodeSerializableValuePolymorphic(Polymorphic.kt:59)
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:36)
	at kotlinx.serialization.json.Json.decodeFromString(Json.kt:100)
	at ru.meanmail.pypi.PyPiKt.loadPackageFromPyPi$lambda-0(PyPi.kt:267)
	at com.intellij.util.io.HttpRequests.doProcess(HttpRequests.java:516)
	at com.intellij.util.io.HttpRequests.process(HttpRequests.java:498)
	at com.intellij.util.io.HttpRequests$RequestBuilderImpl.connect(HttpRequests.java:333)
	at ru.meanmail.pypi.PyPiKt.loadPackageFromPyPi(PyPi.kt:49)
	at ru.meanmail.pypi.PyPiKt.loadPackage(PyPi.kt:143)
	at ru.meanmail.pypi.PyPiKt.getPackageInfo(PyPi.kt:165)
	at ru.meanmail.pypi.PyPiKt.getPackageInfo$default(PyPi.kt:151)
	at ru.meanmail.pypi.PyPiKt.getVersionsList$lambda-6(PyPi.kt:190)
	at com.intellij.openapi.application.impl.ApplicationImpl$2.call(ApplicationImpl.java:326)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:829)
```

**Comment by kostya-ten** [comment]
I confirm there is a problem

**Comment by indepndnt** [comment]
I experienced this bug as well. For me it is `fastapi` and `weasyprint`.

I'm not familiar with Kotlin or this style of stacktrace, but it appears to be complaining specifically about `"comment_text":null` in both cases, which is coming from the package info file from pypi.org. Note below the version and md5 digest in the error snippets match the data I took from the fastapi and weasyprint json's at pypi.org.

```
kotlinx.serialization.json.internal.JsonDecodingException: Unexpected JSON token at offset 189484: Expected string literal but 'null' literal was found.
Use 'coerceInputValues = true' in 'Json {}` builder to coerce nulls to default values.
JSON input: .....l}],"0.75.1":[{"comment_text":null,"digests":{"md5":"c61ad39.....
```
![image](https://user-images.githubusercontent.com/17837901/170875446-0d477425-ef4a-4428-8a0c-082870e99806.png)


```
kotlinx.serialization.json.internal.JsonDecodingException: Unexpected JSON token at offset 79103: Expected string literal but 'null' literal was found.
Use 'coerceInputValues = true' in 'Json {}` builder to coerce nulls to default values.
JSON input: .....ull}],"54.3":[{"comment_text":null,"digests":{"md5":"f7bae96.....
```
![image](https://user-images.githubusercontent.com/17837901/170875492-87292dc9-420f-4763-955a-490c5e99a635.png)

**Comment by shatheeshs** [comment]
Exactly same. confirmation on pycharm 2021.3. Issue is with 'fastapi'
![image](https://user-images.githubusercontent.com/71164322/180593313-66225dfc-1cf8-457f-bf6d-c48023512625.png)

**Comment by make-ing** [comment]
same here... Any news on when we can expect a fix?

**Comment by zeagur** [comment]
confirmed plugin error also occurs on Intellij Ult 2022.2
![image](https://user-images.githubusercontent.com/43737901/184086478-d67931ba-f75e-45c5-b7ab-0985d4f2e6a6.png)

**Comment by eyk107** [comment]
There is allready an PR to fix this issue since three weeks:
https://github.com/meanmail-dev/requirements/pull/90

**Comment by mkniewallner** [comment]
> There is allready an PR to fix this issue since three weeks: #90 @meanmail and @mkniewallner: Could you please check this PR?

Why are you pinging me? I'm not a maintainer of this project.

**Comment by meanmail** [comment]
Fixed in Requirements 2022.3.1-2022.2.2

**Comment by josecsotomorales** [comment]
Thank you @meanmail 🚀
